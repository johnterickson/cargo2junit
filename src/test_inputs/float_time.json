{ "type": "suite", "event": "started", "test_count": 1 }
{ "type": "test", "event": "started", "name": "test_something" }
{ "type": "test", "name": "test_something", "event": "ok", "exec_time": 0.001401918 }
{ "type": "suite", "event": "ok", "passed": 1, "failed": 0, "allowed_fail": 0, "ignored": 0, "measured": 0, "filtered_out": 0 }
