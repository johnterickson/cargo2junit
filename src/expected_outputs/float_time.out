<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite id="0" name="cargo test #0" package="testsuite/cargo test #0" tests="1" errors="0" failures="0" hostname="localhost" timestamp="2023-04-02T23:45:57.758681146Z" time="0.001401918">
    <testcase name="test_something" time="0.001401918" classname="" />
  </testsuite>
</testsuites>
